lib/_tls_wrap.js

```diff
@@ -436,6 +436,11 @@ TLSSocket.prototype._init = function(socket, wrap) {
     self._writableState.errorEmitted = true;
   };
 
+  ssl.onpskexchange = (identity) => {
+    if (options.pskCallback)
+      return options.pskCallback(identity);
```

Comment:
I now assert type is object when truthy. I specifically wanted stuff like `null` and `false` to also work here, in addition to just `undefined`.

Properties are asserted in C++, but need to figure out how to throw exceptions there. (See https://github.com/nodejs/node/pull/6701#discussion_r63226704)